tox.ini

```diff
@@ -68,11 +68,35 @@ commands =
 [testenv:integration]
 description = Run integration tests
 deps =
-    juju
+    juju==2.9.11 # juju 3.3.0 has issues with retrieving action results
     mysql-connector-python
     pytest
     pytest-operator
     pytest-order
     -r{toxinidir}/requirements.txt
 commands =
     pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} --asyncio-mode=auto
+
+[testenv:integration-ha]
+description = Run integration tests
+deps =
+    juju==2.9.11 # juju 3.3.0 has issues with retrieving action results
+    mysql-connector-python
+    pytest
+    pytest-operator
+    pytest-order
+    -r{toxinidir}/requirements.txt
+commands =
+    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} --asyncio-mode=auto -m "ha_tests"
```

Comment:
Paulo had a [good suggestion](https://github.com/canonical/postgresql-operator/pull/10#discussion_r924885319) about moving the asyncio mode to `pyproject.toml` so it's possible to avoid specifying it here in each environment.

Reply:
addressed in 2f6e6e434f2bab7f2a508153eb234fb73edf2bed